src/components/Drawer/drawer.module.scss

```diff
@@ -0,0 +1,38 @@
+@keyframes translate {
+    from { transform: translate(var(--drawer-position-from-x) , var(--drawer-position-from-y)); overflow: auto; }
+    to { transform: translate(var(--drawer-position-to-x) , var(--drawer-position-to-y)); overflow: auto; }
+}
+
+@keyframes translate-reverse {
+    from { transform: translate(var(--drawer-position-from-x) , var(--drawer-position-from-y)); overflow: auto; }
+    to { transform: translate(var(--drawer-position-to-x) , var(--drawer-position-to-y)); overflow: auto; }
+}
```

Comment:
redundant keyframe definition

Reply:
it stops working when translate-reverse is removed